app/src/main/java/com/claraberriel/mvpparking/mvp/view/ParkingSizeView.java

```diff
@@ -0,0 +1,47 @@
+package com.claraberriel.mvpparking.mvp.view;
+
+import android.app.Activity;
+import android.widget.EditText;
+import android.widget.Toast;
+
+
+import com.claraberriel.mvpparking.R;
+import com.claraberriel.mvpparking.databinding.ActivityMainBinding;
+
+import java.util.Objects;
+
+public class ParkingSizeView extends ParkingActivityView {
+    private final ActivityMainBinding binding;
+
+    public ParkingSizeView(Activity activity, ActivityMainBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+    // input to int
+    public int getEdittextNumber() throws NumberFormatException {
+        EditText edittextNumber = binding.edittextMainNumber;
+        return Integer.parseInt(edittextNumber.getText().toString());
+    }
+
+    //reusable
+    public void showToast(String msgName) {
+        Toast.makeText(getContext(), msgName, Toast.LENGTH_LONG).show();
+    }
+
+    public void showParkAmount(int amount) {
+        showToast(Objects.requireNonNull(getContext()).getString(R.string.toast_set, amount));
```

Comment:
`Objects.requireNonNull` will throw an exception when context is not null and that will make your app crash. Replace by `if (context != null)`